openvm/tests/apc_builder_outputs/single_storeh.txt

```diff
@@ -25,7 +25,6 @@ mult=is_valid * 1, args=[write_base_aux__timestamp_lt_aux__lower_decomp__1_0, 12
 // Algebraic constraints:
 flags__1_0 * ((flags__1_0 - 1) * (flags__1_0 - 2)) = 0
 flags__2_0 * ((flags__2_0 - 1) * (flags__2_0 - 2)) = 0
-(flags__1_0 + flags__2_0 + 1) * ((flags__1_0 + flags__2_0) * (flags__1_0 + flags__2_0 - 1)) = 0
```

Comment:
Interesting, this looks like this could be an optimization in openvm itself, or are we using any constants here?